docs/xtrabackup_bin/xbk_option_reference.md

```diff
@@ -132,17 +132,18 @@ with this option enabled. Use at your own risk.
 
 ### --compress()
 This option tells *xtrabackup* to compress all output data, including the
-transaction log file and meta data files, using either the `quicklz` or
-`lz4` compression algorithm. `quicklz` is chosen by default.
+transaction log file and meta data files, using either the `quicklz`, `lz4`, or `ZSTD` compression algorithm. `quicklz` is chosen by default.
 
 When using `--compress=quicklz` or `--compress`, the resulting files have
-the qpress archive format, i.e. every `\*.qp` file produced by *xtrabackup* is
+the qpress archive format. Every `\*.qp` file produced by *xtrabackup* is
 essentially a one-file qpress archive and can be extracted and uncompressed
-by the [qpress](http://www.quicklz.com/) file archiver.
+by the `qpress` file archiver.
 
 `--compress=lz4` produces `\*.lz4` files. You can extract the contents of
 these files by using a program such as `lz4`.
 
+`--compress=zstd` produces `\*.zst` files. You can extract the contents of
+these files by using the `--decompress` option.
```

Comment:
forgot to mention, ZSTD also adds a new option to adjust compression level:

```
  --compress-zstd-level=# 
                      Zstandard compression level. from 1 - 19. The default
                      value is 1.
```

Reply:
Updated

Reply:
Thanks!